bitgo-java-api/src/main/java/com/bitso/bitgo/v2/BitGoClient.java

```diff
@@ -66,11 +67,11 @@
      *
      * @param coin
      * @param walletId
-     * @param walletTransferId
+     * @param txid
      * @return
      * @throws IOException
      */
-    Optional<Map<String, Object>> getWalletTransferId(String coin, String walletId, String walletTransferId) throws IOException;
+    Transfer getWalletTransferId(String coin, String walletId, String txid) throws IOException;
```

Comment:
Shouldn't this be an Optional&lt;Transfer&gt;?

Reply:
done